src/platform-includes/sourcemaps/artifact-and-release-bundles/javascript.mdx

```diff
@@ -0,0 +1,63 @@
+This article explains the differences between Artifact and Release Bundles, and the implications of using them. Visit [Uploading Source Maps](/platforms/javascript/guides/react/sourcemaps/uploading/) if you're looking for our guides on how to upload source maps.
+
+## Introduction
+
+Sentry currently supports two methods of uploading sourcemaps: **Release Bundles** and **Artifact Bundles**. A bundle, at a higher level, contains all the minified sources and source maps (known as _artifacts_) required for Sentry to unminify the frames of a stack trace.
+
+## Release Bundles
+
+### What Are They?
+
+A release bundle is a bundle format that is connected to an existing `release` in Sentry. For this reason **the creation of a `release` is required** before uploading your artifacts.
+
+| Pros | Cons |
+|:--|:--|
+| Works with every SDK version | Configuring your build pipeline to upload artifacts can be very challenging  |
+| | Requires your project to use Sentry Releases
+
+### How Do They Work?
+
+```
+Diagram Here
+```
+
+Release Bundles are called like that because they use a `release` as way to index a specific bundle*. The `release` is extracted from the incoming error and used to lookup the most recent bundle. When a bundle is found, we use the absolute path in each frame to find the corresponding minified source in the retrieved bundle. Once the minified source is found, we try to discover its source map with a series of heuristics (e.g., looking at the `sourceURLMapping` comment, using metadata from the uploaded bundle).
+
+In practice, matching the paths for artifacts discovery is prone to mistmatchs that might be caused by a wrong configuration of your bundler, CI/CD pipeline...
+
+## Artifact Bundles
+
+### What Are They?
+
+An artifact bundle is our new bundle format that aims to address the shortcomings of release bundles by **adding the support for Debug IDs** and **removing the need to create a `release`**.
+
+| Pros | Cons |
+|:--|:--|
+| With the use of Debug IDs, configuring your build pipeline to upload artifacts is trivial | Requires a version update to Javascript SDKs, bundler plugin ( and `sentry-cli`) |
+| Associating a release is still possible, but not required | May cause problems with integrity checks
+
+### How Do They Work?
+
+```
+Diagram Here
+```
+
+Artifact Bundles solve the challenge of matching paths by using a new identification mechanism known as Debug IDs. Rather than relying on paths we uniquely identify and bind pairs of minified sources and source maps with an Debug ID(UUID). With this, we are able to quickly identify the minified source and corresponding source map without ever needing to check any paths. Thus, you don't have to worry about wrong paths configurations.
+
+<Note>
+
+If you want to learn more about the rationale behind Debug IDs, we suggest taking a look at our [engineering blog](https://sentry.engineering/blog/the-case-for-debug-ids).
+
+</Note>
```

Comment:
```suggestion
Artifact bundles solve the challenge of matching paths by using a new identification mechanism known as Debug IDs. So rather than having to uniquely identify paths then bind pairs of minified sources and source maps with a Debug ID(UUID), you can quickly identify the minified source and corresponding source map without ever needing to check any paths. Thus, incorrect path configurations no longer present a problem.

<Note>

To learn more about the rationale behind Debug IDs, check out our [engineering blog](https://sentry.engineering/blog/the-case-for-debug-ids).

</Note>
```